lib/src/common.dart

```diff
@@ -46,4 +46,5 @@ export 'package:github/src/common/util/oauth2.dart';
 export 'package:github/src/common/util/pagination.dart';
 export 'package:github/src/common/util/service.dart';
 export 'package:github/src/common/util/utils.dart';
+export 'package:github/src/const/language_color.dart';
```

Comment:
Just a note, but now that this file is being exported, the language color map is now considered part of the public API and should be non-breaking in future changes.

Reply:
Yes!